index.markdown

```diff
@@ -621,6 +621,11 @@ Capital One credit card that's directly targeted for people with no credit
 history/newcomers. Beware this card is free for the first year, but costs around
 $30 per year after that.
 
+#### Wells Fargo
+
+Stefan got declined twice trying to get a secured credit card there. Both times they 
+said my SSN was invalid. If you want a CC, wouldn't recommend Wells Fargo.
```

Comment:
I entered it online, and once on the phone. My SSN has no credit history nor do I have any credit cards associated with it.